installer/win/Brackets_en-us.wxl

```diff
@@ -55,4 +55,6 @@
     <String Id="VerifyReadyDlgInstallText" Overridable="yes">Click Install to begin, or Cancel to exit.</String>
     <String Id="VerifyReadyDlgRemoveText" Overridable="yes">Click Remove to uninstall !(loc.ShortProductName), or Cancel to exit.</String>
     <String Id="OpenWithText" Overridable="yes">Open With !(loc.ShortProductName)</String>
+    <String Id="AddBracketsToPath" Overridable="yes">Update PATH environment variable with !(loc.ShortProductName) install location (in order to launch !(loc.ShortProductName) from command line)</String>
+    <String Id="AddContextMenu" Overridable="yes">Add &quot;Open With !(loc.ShortProductName) &quot; menu to file and folder context menus</String>
```

Comment:
Suggested string changes:

> Add "brackets" launcher shortcut to PATH for command line use
> Add "Open with Brackets" to Explorer context menu for all files and folders

Two implications of this:

* Add a new variable `loc.ProductCommandName` for the lowercase "brackets" string
* Change the "OpenWithText" string to use lowercase "with," as seen in the 2nd string above -- that's consistent with other apps and Windows Explorer itself.

But I also think we should split "OpenWithText" into two strings for files & folders:

> Open with Brackets
> Open as Brackets project

This is easy to do and adds some clarity.

Lastly, while we're splitting it, I'd suggest we use string key names that are harder to confuse with the _regular_ "Open With" menu, which the installer also deals with.  Maybe "ExplorerMenuOpenFile" & "ExplorerMenuOpenFolder"?